packages/direflow-component/src/helpers/polyfillHandler.ts

```diff
@@ -26,9 +26,12 @@ const includePolyfills = async (
   let useSD = '';
   let useCE = '';
   let useAD = '';
-
   const polyfillLoaderPlugin = plugins?.find((plugin) => plugin.name === 'polyfill-loader');
 
+  const disableSD = typeof polyfillLoaderPlugin?.options?.use.sd === 'boolean' && !polyfillLoaderPlugin?.options?.use.sd;
+  const disableCE = typeof polyfillLoaderPlugin?.options?.use.ce === 'boolean' && !polyfillLoaderPlugin?.options?.use.ce;
+  const disableAD = typeof polyfillLoaderPlugin?.options?.use.adapter === 'boolean' && !polyfillLoaderPlugin?.options?.use.adapter;
```

Comment:
maybe this would be a little bit more readable in my opinion? :)
instead of checking if boolean and if not true. just check if false

```suggestion
  const disableSD = polyfillLoaderPlugin?.options?.use.sd === false;
  const disableCE = polyfillLoaderPlugin?.options?.use.ce === false;
  const disableAD = polyfillLoaderPlugin?.options?.use.adapter === false;
```